### PR TITLE
[stable/kubewatch] Add more resources for watch

### PR DESCRIPTION
**Pull request by oleg-filiutsich**
#### What this PR does / why we need it:
Add more resources for wath
```
  ingress: false
  namespace: false
  secret: false
  configmap: false
```

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/kubewatch/README.md [hunk 1]
```diff
@@ -87,6 +87,11 @@ The following table lists the configurable parameters of the kubewatch chart and
 | `resourcesToWatch.services`              | watch changes to [Services](https://kubernetes.io/docs/concepts/services-networking/service/)              | `false`                           |
 | `resourcesToWatch.job`                   | watch changes to [Jobs](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/) | `false`                           |
 | `resourcesToWatch.persistentvolume`      | watch changes to [PersistentVolumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/)      | `false`                           |
+| `resourcesToWatch.ingress`      | watch changes to [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/)      | `false`                           |
+| `resourcesToWatch.namespace`      | watch changes to [Namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/)      | `false`                           |
```
Comment (juan131) [inline stable/kubewatch/README.md:91]: What's the difference between this parameter and the parameter `namespaceToWatch`?

File: stable/kubewatch/values.yaml [hunk 2]
```diff
@@ -45,6 +45,10 @@ resourcesToWatch:
   pod: true
   job: false
   persistentvolume: false
+  ingress: false
+  namespace: false
+  secret: false
+  configmap: false
```
Comment (juan131) [inline stable/kubewatch/values.yaml:53]: Please update the `values-production.yaml` too

File: stable/kubewatch/Chart.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 name: kubewatch
-version: 0.8.12
+version: 0.8.13
```
Comment (juan131) [inline stable/kubewatch/Chart.yaml]: Please bump minor version instead.

**Review by juan131 — CHANGES_REQUESTED** [review]
Hi @oleg-filiutsich 

Thanks so much for your PR! Please take a look to my suggestion when you have a chance.

File: stable/kubewatch/README.md (hunk 1 above)
Comment (oleg-filiutsich) [reply stable/kubewatch/README.md:91]: Hi @juan131 .
**namespaceToWatch** configure kubewatch monitoring resources (deployment,pod and etc.) only in specific namespace (as space for resources).
**resourcesToWatch.namespace** tell kubewatch catch changes in create/reconfigure namespaces (as object) in all cluster

File: stable/kubewatch/values.yaml (hunk 2 above)
Comment (oleg-filiutsich) [reply stable/kubewatch/values.yaml:53]: done

Comment (oleg-filiutsich) [reply stable/kubewatch/values.yaml:53]: done

File: stable/kubewatch/Chart.yaml (hunk 3 above)
Comment (oleg-filiutsich) [reply stable/kubewatch/Chart.yaml]: please check again

**Pull request closed without merging**